modules/statistics/RTPStatsCollector.js

```diff
@@ -401,6 +404,27 @@ StatsCollector.prototype._defineGetStatValueMethod = function(keys) {
     return (item, name) => itemStatByKey(item, keyFromName(name));
 };
 
+/**
+ * Obtains a stat value from given stat and converts it to a non-negative
+ * number. If the value is either invalid or negative then 0 will be returned.
+ * @param report
+ * @param {string} name
+ * @return {number}
+ */
+StatsCollector.prototype.getNonNegativeStat = function(report, name) {
+    let value = this._getStatValue(report, name);
```

Comment:
> That number is larger than 2147483647 (2^32 -1) so the code is actually working even better, since that's not a valid SSRC number, isn't it?

The number you quote is actually 2^31 - 1, and 2290353494 is a valid value for an SSRC . Also, there's no mention that this necessarily only reads SSRC values, and the documentation explicitly says it returns 0 if the value is invalid.